client/state/audio/middleware.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { has } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	HAPPYCHAT_RECEIVE_EVENT,
+} from 'state/action-types';
+
+export const playSound = src => {
+	if ( typeof Audio !== 'function' ) {
+		// No Audio support in this browser
+		return;
+	}
```

Comment:
I think I lost a comment here so I'm going to repeat it and hope I'm not duplicating/nagging you.

what is actually going on here if this condition evaluates false?

we're not actually wanting to do anything and it's not like we boot up the app _with_ audio support then at some point in the runtime lose it.

while this is a great check _to keep in here_ I think we could benefit by putting two additional checks purely for safety. one at the point where we actually inject the middleware (no need to inject it if no `Audio` is available) and the other is in the default export below. if for some reason we accidentally inject it we can skip initializing the middleware completely.

@blowery has redone the middleware chain in #10897 and this should probably wait to rebase off of those changes